hcl2template/types.packer_config.go

```diff
@@ -110,8 +110,9 @@ func (cfg *PackerConfig) EvalContext(ctx BlockContext, variables map[string]cty.
 			}),
 			buildAccessor: cty.UnknownVal(cty.EmptyObject),
 			packerAccessor: cty.ObjectVal(map[string]cty.Value{
```

Comment:
We can remove the accessor here since we assign it later, it doesn't need to be added as part of the initial declaration

Reply:
Yeah nevermind, you addressed it before I pushed my comment :D

We can merge